docs/connector-development/README.md

```diff
@@ -117,7 +117,7 @@ In order to enable normalization for a destination connector, you'll need to set
 ### New connectors
 If you're adding normalization to a new connector, you'll need to first add a destination definitions entry:
 1. Add a new connector definition in `airbyte-config-oss/init-oss/src/main/resources/seed/destination_definitions.yaml`. You can copy an existing entry and modify it to match your connector, generating a new UUIDv4 for the `destinationDefinitionId`.
-2. Run the command `./gradlew :airbyte-config:init:processResources` to generate the seed spec yaml files, and commit the changes to the PR. See [this readme](https://github.com/airbytehq/airbyte/tree/master/airbyte-config-oss/specs) for more information.
+2. Run the command `./gradlew :airbyte-config:init:processResources` to generate the seed spec yaml files, and commit the changes to the PR. See [this readme](https://github.com/airbytehq/airbyte/tree/master/airbyte-config-oss/specs-oss) for more information.
```

Comment:
@erohmensing I found the link was broken here too, and figured that the one you updated yesterday should point to the same live page rather than a static page that may not reflect any future updates.